Makefile

```diff
@@ -15,14 +16,15 @@ help:
 	@echo "stop              Stops the Docker container."
 	@echo ""
 	@echo "Variables:"
-	@echo "SIMFILE_PATH       Path to the sim.json file."
+	@echo "SIMFILE_PATH      Path to the sim.json file."
 	@echo "LOG_LEVEL         Set the logging level (default: info) e.g. <make run LOG_LEVEL=debug>"
 	@echo "HELP              Set to true to print the help message (default: false) e.g. <make run HELP=true>"
 	@echo "PRINT_BATCH_SIZE  Set the batch size for printing the results e.g. <make run PRINT_BATCH_SIZE=100>"
 	@echo "TOTAL_TIME        Set the total time for the simulation e.g. <make run TOTAL_TIME=1000>"
+	@echo "DATA_DIR          Set the data directory for the simulation containing simulation files and results e.g. <make run-docker DATA_DIR="/Users/anon/data>""
 
 run-docker:
-	docker run -d --rm --name sim-ln --init -v simln-data:/data -e SIMFILE_PATH=/data/sim.json -e LOG_LEVEL=$(LOG_LEVEL) -e HELP=${HELP} -e PRINT_BATCH_SIZE=${PRINT_BATCH_SIZE} -e TOTAL_TIME=${TOTAL_TIME} sim-ln
+	docker run -d --rm --name sim-ln --init -v simln-data:/data -e SIMFILE_PATH=/data/sim.json -e DATA_DIR=${DATA_DIR} -e LOG_LEVEL=$(LOG_LEVEL) -e HELP=${HELP} -e PRINT_BATCH_SIZE=${PRINT_BATCH_SIZE} -e TOTAL_TIME=${TOTAL_TIME} sim-ln
 
 run-interactive:
```

Comment:
I didn't catch this when it was added, but I'm wondering now:

Don't we want to also pass the env variables when running interactively?

Reply:
hmm... we should. I'll fix that.

Reply:
fixed here -> https://github.com/bitcoin-dev-project/sim-ln/pull/159/commits/85082b3b80ac1299cf750a239544594b2ac2df3e